aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -1046,7 +1046,7 @@ std::tuple<Tensor, Tensor, Tensor> _svd_helper_cpu(const Tensor& self, bool some
 
     if (compute_uv) {
       if (some) {
-        VT_working_copy = VT_working_copy.narrow(-1, 0, k);
+        VT_working_copy = VT_working_copy.narrow(-2, 0, k);
```

Comment:
With these changes to _svd_helper, is there any change in behavior to torch.svd?

Reply:
`torch.svd` behavior is unchanged: this line was introduce by commit a14c2fb, which fixes `_create_U_S_VT`, and `narrow(-2)` instead of `-1` is needed because the `VT` matrix is now transposed compared to the old (wrong) one.

To be super-careful, I just tried to revert this change and the old svd tests started failing, so I am pretty sure this is correct.